packages/framework/src/servers/nest/nest.module-definition.ts

```diff
@@ -0,0 +1,17 @@
+import { ConfigurableModuleBuilder } from '@nestjs/common';
+import { NovuModuleOptions } from './nest.interface';
+
+// use ConfigurableModuleBuilder, because building dynamic modules from scratch is painful
```

Comment:
Following best practices to use ConfigurableModuleBuilder per [Dynamic Modules docs](https://docs.nestjs.com/fundamentals/dynamic-modules#configurable-module-builder)